tensorflow/models/embedding/word2vec.py

```diff
@@ -378,7 +378,7 @@ def save_vocab(self):
     opts = self._options
     with open(os.path.join(opts.save_path, "vocab.txt"), "w") as f:
       for i in xrange(opts.vocab_size):
-        f.write("%s %d\n" % (tf.compat.as_text(opts.vocab_words[i]),
+        f.write("%s %d\n" % (tf.compat.as_text(opts.vocab_words[i]).encode('utf-8'),
```

Comment:
Please fix the indentation here. "(tf.compat..." should be justified with ""%s..." and make sure that the line length is < 80.